plugwise/helper.py

```diff
@@ -445,14 +445,17 @@ def _appliance_info_finder(self, appliance, appl):
             appl.model = appl.name = self.smile_name
             appl.v_name = "Plugwise B.V."
 
-            # Adam: check for cooling capability, assume heating capability is always present
+            # Adam: check for cooling capability, and check if active
```

Comment:
`# Adam: check for cooling capability and state` ?

Reply:
and maybe add cross comments that Anna is determined through appliance data (and vv)?